libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -778,11 +778,6 @@ class LibCLCRemangler : public ASTConsumer {
   void Initialize(ASTContext &C) override {
     ASTCtx = &C;
     SMDiagnostic Err;
-#if SPIRV_ENABLE_OPAQUE_POINTERS || !defined(__SPIR__)
-    LLVMCtx.setOpaquePointers(true);
-#else
-    LLVMCtx.setOpaquePointers(false);
```

Comment:
The default is that context has it set to `false`, so there is no need for `setOpaquePointers`, right?